charts/pvc-autoresizer/templates/controller/serviceaccount.yaml

```diff
@@ -1,7 +1,10 @@
+{{- if .Values.serviceAccount.create }}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ template "pvc-autoresizer.fullname" . }}-controller
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "pvc-autoresizer.labels" . | nindent 4 }}
+automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
```

Comment:
We might not need this field if we add `.Values.serviceAccount.enabled`, since users can create their own custom service accounts. What do you think?

Reply:
Personally, I always appreciate when I can use the chart resources provided and customize them with the values. Otherwise sometimes it can be extra work to replicate things like the name, etc. that other resources expect. That said, I figured there may be other users who want to use a completely custom service account. Unless you all have a strong preference, I would prefer to keep this field. Let me know if you have any thoughts!

Reply:
I got it. Thank you for your clarification!